test/parallel/test-permission-deny-fs-symlink.js

```diff
@@ -0,0 +1,98 @@
+// Flags: --experimental-permission --allow-fs-read=* --allow-fs-write=*
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const fs = require('fs');
+const fixtures = require('../common/fixtures');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const blockedFile = fixtures.path('permission', 'deny', 'protected-file.md');
+const blockedFolder = tmpdir.path + '/';
+const regularFile = __filename;
+const symlinkFromBlockedFile = 'example-symlink.md';
+
+{
+  // Symlink previously created
+  fs.symlinkSync(blockedFile, symlinkFromBlockedFile);
+  assert.ok(process.permission.deny('fs.read', [blockedFile, blockedFolder]));
+  assert.ok(process.permission.deny('fs.write', [blockedFile, blockedFolder]));
+}
+
+{
+  // Previously created symlink are NOT affected by the permission model
+  fs.readlink(symlinkFromBlockedFile, (err) => {
+    assert.ifError(err);
+  });
```

Comment:
what about reading the file that the symlink points to?

Reply:
It was kind of a rhetorical question. Can we also test it (`fs.readFile(symlink)`) rather than just document it?

Reply:
Fixed in 7dc06682d5